packages/rs-dpp/src/data_contract/v0/methods/schema.rs

```diff
@@ -60,13 +60,13 @@ impl DataContractMethodsV0 for DataContractV0 {
             .collect()
     }
 
-    fn schema_defs(&self) -> Option<&BTreeMap<DefinitionName, Value>> {
+    fn schema_defs(&self) -> Option<&Value> {
```

Comment:
This will change lookup time from 1 to N

Reply:
Lookup happens only in try_from_schema. We convert it to a map as far as I remember. But for user it's more convenient to deal with value since he can do:
```
let def = platform_value!({
   "a": {
        "type": "string"
    }
})

let doc1 = platform_value!({
  "properties": {
      $ref: "#/$defs"
   }
})
```

Reply:
I don't understand.